src/utils.js

```diff
@@ -486,26 +500,28 @@ export function processConfig (data, userList, preferences, platformSpecificFeat
         return feature.state === 'enabled' && !isUnprotectedDomain(topLevelHostname, feature.exceptions)
     }).concat(platformSpecificFeaturesNotInRemoteConfig) // only disable platform specific features if it's explicitly disabled in remote config
     const isBroken = isUnprotectedDomain(topLevelHostname, data.unprotectedTemporary)
-    preferences.site = {
+    /** @type {Record<string, any>} */
+    const output = Object.assign({}, preferences)
```

Comment:
not a blocker of course, but I wouldn't default to `Object.assign({}, foo)` for this case - just because it's a mutating operation on the first arg and easy to get wrong.  

I'd do this instead:

```js
const output = { ...preferences } 
```